src/Components/AgeCheck/AgeCheck.jsx

```diff
@@ -0,0 +1,33 @@
+import './AgeCheck.scss';
+import { useState } from 'react';
+
+function AgeCheck({ setisLegal }) {
+  
+  const ageVerification = (canAccessSite) => {
```

Comment:
Looks like theres some left over code here from having two buttons. We can go ahead and remove that canAccessSite param and argument!